samples/jsglobal.ts

```diff
@@ -15,3 +15,7 @@ declare module nested {
         static isCirce(thing: any): boolean;
     }
 }
+
+declare const globalConst: String;
+declare let globalVar: String;
+declare function globalFunc(): String;
```

Comment:
Please make sure that the file ends with a new line, so that git/GitHub does not display that ugly mark.

Reply:
Done